hw/ip/aon_timer/data/aon_timer.hjson

```diff
@@ -95,6 +95,43 @@
       width:   "1"
     }
   ],
+  features: [
+    { name: "AON_TIMER.WAKEUP",
+      desc: '''
+            The AON_TIMER tick period can be configured.
+            '''
+    },
+    { name: "AON_TIMER.WATCHDOG.BARK",
+      desc: '''
+            The AON_TIMER block has a watchdog that can bark if not petted in the programed time.
+            '''
+    },
+    { name: "AON_TIMER.WATCHDOG.BITE",
+      desc: '''
+            The AON_TIMER block has a watchdog that can bite causing a reset if not petted in the programed time.
+            '''
+    },
+    { name: "AON_TIMER.WATCHDOG.PAUSE",
+      desc: '''
+            The AON_TIMER block will pause the watchdog when the heart is in sleep mode.
```

Comment:
I'm not sure I understand "heart" here. This state comes from a `sleep_mode_i` top-level port, so maybe something like "... when told the chip is in sleep mode" or similar?

But I think this feature is configurable by a register? (WDOG_CTRL). Maybe this feature needs rewording a bit to mention that it doesn't always work like that.

Reply:
Thanks for the catch, I rephrased it.